satpy/tests/reader_tests/test_seviri_l2_grib.py

```diff
@@ -0,0 +1,185 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2019 Satpy developers
+#
+# satpy is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# satpy is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with satpy.  If not, see <http://www.gnu.org/licenses/>.
+
+"""The seviri_l2_grib reader tests package."""
+
+import numpy as np
+import datetime
+
+from satpy import CHUNK_SIZE
+from satpy.readers.seviri_l2_grib import (SeviriL2GribFileHandler,
+                                          REPEAT_CYCLE_DURATION)
+
+import unittest
+
+try:
+    from unittest import mock
+except ImportError:
```

Comment:
Since py27 is not supported any more, this can be removed to just say `from unittest import mock` (no try/except).

Reply:
removed.